beanstalk_worker/Dockerfile.build_worker

```diff
@@ -5,7 +5,7 @@ RUN yum update -y && \
     yum clean all
 
 RUN mkdir -p /run_worker
-ADD beanstalkc.py ca.crt node.kubeconfig worker_start_build.py config.py /run_worker/
+ADD beanstalkc.py ca.crt node.kubeconfig worker_start_build.py config.py lib.py /run_worker/
```

Comment:
I am assuming the lib.py is a mock of some sort as its not in repo right now, nor is it in this PR.